### PR TITLE
Potent Hams

### PR DESCRIPTION
**Pull request by Arthurdentist**
Adds Potent Ham as a food recipe.

To make one you need one Asimov Core Module, 10 units of Plasma and a Roburger, pretty much just a fancy Roburger.

potent hams

Its been a long time since I have contributed anything so please enjoy my bottom of the barrel, absolutely zero effort waste of space non-addition.

### CODE REVIEW

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -3475,3 +3475,12 @@
...
 			jump()
 	return 1
+
+/obj/item/weapon/reagent_containers/food/snacks/potentham
+	name = "potent ham"
+	desc = "I'm sorry Dave, but I am afraid I can't let you eat that."
+	icon_state = "potentham"
+	New()
+		..()
+		reagents.add_reagent("nanites", 2)
```
Comment (SkowronX) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: WOAH HAMBURGERS

**Comment by Exxion** [comment]
What's with all the changed lines that don't actually appear to be changed?

**Comment by Arthurdentist** [comment]
Not sure, haven't touched any of them. Might just be byond being shit but as far as I can tell it all works fine.

**Comment by Kurfursten** [comment]
Instead of Nanites, I would give it flags = NOREACT and the ingredients for a small EMP, so it triggers a tiny EMP when you bite it.

Because Potent Hams shut down machines.

**Comment by Arthurdentist** [comment]
thats not a bad idea Kurfursten, I'll consider it.

**Comment by Midaychi** [comment]
Downside of giving it noreact is then you basically turn it into a 50 unit cryostasis beaker

**Comment by Kurfursten** [comment]
It is actually probably easier to get a cryostasis beaker than the ingredients for this food.

**Comment by Arthurdentist** [comment]
could try and code a new chemical which creates an EMP on consumption only.

**Comment by Kurfursten** [comment]
Wouldn't be hard, add on_mob_life to the reagent, then empulse(M.loc,0,1,0)

That would make the imbiber throw off a tiny EMP every tick. 0 heavy range, 1 light range.

**Comment by Arthurdentist** [comment]
added a simple EMP chemical and changed the recipe from a roburger to a robotic head and a pair of handcuffs

**Comment by Arthurdentist** [comment]
changed the chemical based on Clusterfack's suggestions in IRC. Now it will only activate on biting into the ham instead of EMPing each tick, I made the EMP slightly larger seeing as how its only going to fire once now. Everything is working fine and this should be ready to merge unless i've missed something.

**Pull request merged**